package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "todo-list-app",
+  "version": "1.0.0",
+  "description": "A todo-list app for adding and removing items",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack --config webpack.config.js"
+  },
```

Comment:
- [ ] Kindly make sure your dev-server is well configured because when I start the server. The process quits without rendering the intended page. Please implement this change as requested by the previous reviewer 🙏